resources/views/staff/circles/edit.blade.php

```diff
@@ -4,42 +4,62 @@
 
 @section('content')
 <div class="container">
-    <form method="post" action="#">
+    <form method="post" action="{{ route('staff.circles.store', $circle) }}">
         @csrf
-        <div class="form-group row">
-            <label for="nameInput" class="col-sm-2 col-form-label">団体名</label>
-            <div class="col-sm-10">
-                <input
-                    id="nameInput"
-                    class="form-control"
-                    type="text"
-                    name="name"
-                    value="{{ old('name', $circle->name) }}"
-                    required
-                >
-            </div>
-        </div>
+        <div class="card">
+            <div class="card-body">
+                <div class="form-group row">
+                    <label for="nameInput" class="col-sm-2 col-form-label">団体名</label>
+                    <div class="col-sm-10">
+                        <input
+                            id="nameInput"
+                            class="form-control @if ($errors->has('name')) is-invalid @endif"
```

Comment:
@if を使うより、こういう書き方をした方が良いかなと思ったのですが、どうでしょう…？

```suggestion
                            class="form-control {{ $errors->has('name') ? 'is-invalid' : '' }}"
```